All Core Devs Meetings/Meeting 59.md

```diff
@@ -0,0 +1,451 @@
+# Ethereum Core Devs Meeting 59 Notes
+### Meeting Date/Time: Friday, April 12, 2019 14:00 UTC
+### Meeting Duration: 1 hours 30 minutes
+### [GitHub Agenda Page](https://github.com/ethereum/pm/issues/93)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=gfC92gQKKnI)
+
+**Tim**: I welcome everyone on Core Dev Meeting # 59 I'm Tim not Hudsun and I will be facilitating.
+# 1. [Review previous decisions made and action items](https://github.com/ethereum/pm/issues/93)
+**Tim**: Let's go through previous decisions and action items. Going through agenda.
+
+**DECISION 58.1**: Stop [ProgPoW Carbon Vote](http://www.progpowcarbonvote.com).
+
+**Tim** :  Last block number for that is 7504000. If people want to vote on the carbon votes, they needed to leave there Ether in whatever address to use to vote until that block. And then their will be a snapshot taken and then people can withdraw there Ether. Does anyone have anything to add there?
+**Pooja**:  Yeah I think the last date was a 10th of April only and is closed now says the website.
+
+**Status**: ProgPoW Carbon voting is closed. 
+
+**DECISION 58.2**: Clients status update will be provided as comments in the agenda.
+
+**Tim**: I'm going to go over every single client to research group and if people have questions, add them on the agenda and we'll make sure that it stays for people that asked question.
+
+**Status**: Done
+
+**DECISION 58.3**: Reseach will be updated in the comments in the agenda.
+
+**Status**: Done
+
+**ACTION 58.1** Cat Herders to look at updating EIP1.
+
+**Tim**: I know there's been some stuff happening I've personally spent a lot of time there trying to describe the process and sort of gather all the various conversations that have happened around this. I don't know if anyone has anything to add. 
+
+**Status**: WIP
```

Comment:
Not sure if we want to add it, but the associated GH issue is https://github.com/ethereum-cat-herders/PM/issues/19